nixos/modules/programs/wayland/wayfire.nix

```diff
@@ -46,5 +47,10 @@ in
       # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1050914
       config.wayfire.default = lib.mkDefault [ "wlr" "gtk" ];
     };
-  };
+  }
+  (import ./wayland-session.nix {
```

Comment:
Can we use the module system here?

`imports = [ ]` in top level?

Reply:
i dont think we can do that without turning wayland-session.nix into an actual module, instead of a function, and modifying all callers.
This was just a quick fix emulating the design choices by the maintainers of the wayland modules